great_expectations/render/types/__init__.py

```diff
@@ -501,44 +508,91 @@ class RenderedAtomicValue(DictDot):
     def __init__(
         self,
         schema: Optional[dict] = None,
+        header: Optional["RenderedAtomicValue"] = None,
         template: Optional[str] = None,
         params: Optional[dict] = None,
-        header: Optional["RenderedAtomicValue"] = None,
         header_row: Optional[List["RenderedAtomicValue"]] = None,
         table: Optional[List[List["RenderedAtomicValue"]]] = None,
         graph: Optional[dict] = None,
         kwargs: Optional[dict] = None,
     ) -> None:
         self.schema: Optional[dict] = schema
+        self.header: Optional[RenderedAtomicValue] = header
 
         # StringValueType
         self.template: Optional[str] = template
         self.params: Optional[dict] = params
 
         # TableType
-        self.header: Optional[RenderedAtomicValue] = header
         self.header_row: Optional[List[RenderedAtomicValue]] = header_row
         self.table: Optional[List[List[RenderedAtomicValue]]] = table
 
         # GraphType
-        self.graph: Optional[RenderedAtomicValue] = graph
+        self.graph: Optional[RenderedAtomicValueGraph] = RenderedAtomicValueGraph(
+            graph=graph
+        )
 
         # UnknownType
         self.kwargs: Optional[dict] = kwargs
 
+    def __repr__(self) -> str:
+        return json.dumps(self.to_json_dict(), indent=2)
+
+    def __str__(self) -> str:
+        return json.dumps(self.to_json_dict(), indent=2)
+
+    def to_json_dict(self) -> dict:
+        """Returns RenderedAtomicValue as a json dictionary."""
+        d = renderedAtomicValueSchema.dump(self)
```

Comment:
`None` attributes will be dropped during this step and will only be added to the dictionary below if they are found.